src/engine/source/base/parseEvent.cpp

```diff
@@ -70,7 +70,7 @@ Event parseOssecEvent(const std::string& event)
 
     const bool isFullLocation = (FIRST_FULL_LOCATION_CHAR == event[2]);
 
-    const auto secondColonIdx {event.find(":", 2)};
+    const auto secondColonIdx {event.find(':', 2)};
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

'find' called with a string literal consisting of a single character; use the more effective overload accepting a character